augur/tasks/git/dependency_tasks/dependency_util/python_deps.py

```diff
@@ -2,17 +2,32 @@
 import re
 from pathlib import Path
 import codecs
+import ast
```

Comment:
I think we need to add this to the setup.py for the task. @ABrain7710 / @IsaacMilarky ?

Reply:
ast is builtin